documentation/adr/0002-use-github-for-documentation.md

```diff
@@ -16,14 +18,21 @@ Documentation will directly reference Horace and Herbert projects.
 
 The GitHub Wiki repository does not support pull requests, or the display of any content not on the the `master` branch.
 
+
 ## Decision
 
 We will store documentation in the `Horace/documentation` folder and link to this from the the [GitHub Wiki](https://github.com/pace-neutrons/pace-developers/wiki) which is attached to the PACE project on GitHub. 
 
+The documentation will be supported and built by the Jenkins pipeline and automatically deployed on:
+- nightly builds (unstable)
+- release pipeline (stable & horace-version)
+
+The Jenkins pipeline will also build the documentation as a zipped artifact for examination on other builds.
```

Comment:
@oerc0122 Not clear what examination on other builds means. What workflow would this imply? I was presuming that zipped artefacts could be unzipped at remote locations for easy access by users also downloading Horace/PACE.